include/oneapi/dpl/execution

```diff
@@ -67,17 +67,6 @@ _PSTL_PRAGMA_MESSAGE_POLICIES(
     "The <Parallel STL> execution policies are injected into the standard namespace std::execution")
 #endif // _ONEDPL_CPP17_EXECUTION_POLICIES_PRESENT
 
-namespace oneapi
```

Comment:
This namespace deleted here because it's already created in file execution_defs.h